### ISSUE TITLE
SetAnimationByName does not work

### ISSUE THREAD

**Issue by zeelorenc** [ziggi/FCNPC#32]
I have put one NPC inside interior. When I enter the interior, the animation is not synced!

```pawn
public FCNPC_OnSpawn( npcid )
{
    FCNPC_ApplyAnimation( npcid, "GANGS", "null", 0.0, 0, 0, 0, 0, 0 ); // preload
    FCNPC_ApplyAnimation( npcid, "GANGS", "leanIDLE", 3.0, 0, 1, 1, 1, 0 );
    FCNPC_SetAnimationByName( npcid, "GANGS:leanIDLE", 3.0, 0, 1, 1, 1, 0 );
    return 1;
}
```

Here's my code. Cheers.

**Comment by Crayder** [comment]
Preloading animations is for the person viewing the animation, not the person doing the animation.

No matter what, if an animation is applied to a player/NPC, that player/NPC tells the server it's doing an animation. If any players are around, the server tells them too. When players are told about other players/NPCs doing animation it's played for them on their end. If they don't have it preloaded on THEIR end then it might not play.

Basically... preloading animations for NPC's is 100% pointless because they are not going to see the animation. You only have to preload animations for people that are going to see the animation.

**Comment by zeelorenc** [comment]
Pointless or not, the issue is not addressed.

**Comment by Crayder** [comment]
The issue is not addressed?

>You only have to preload animations for people that are going to see the animation.

**Not** the NPC.

**Comment by ziggi** [comment]
Fixed.
